LoopKitUI/Views/Information Screens/GlucoseTherapySettingInformationView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  GlucoseTherapySettingInformationView.swift
+//  LoopKitUI
+//
+//  Created by Rick Pasetto on 11/16/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+import SwiftUI
+import LoopKit
+
+public struct GlucoseTherapySettingInformationView: View {
+    var text: AnyView!
+    let onExit: (() -> Void)?
+    let mode: PresentationMode
+    let therapySetting: TherapySetting
+    let preferredUnit: HKUnit
+    
+    @Environment(\.presentationMode) var presentationMode
+    
+    public init(
+        therapySetting: TherapySetting,
+        preferredUnit: HKUnit? = nil,
+        text: AnyView? = nil,
+        onExit: (() -> Void)?,
+        mode: PresentationMode = .acceptanceFlow
+    ){
+        self.therapySetting = therapySetting
```

Comment:
[nit] Order of property declarations above, function parameters above, and property assignments here are all different. Consider making the order the same, if possible. It is easier to grok all of the properties together if the order is similar.